src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,119 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeException;
+
+/**
+ * Type of task which includes date.
+ */
+public class Event extends Task {
+    private String date;
+
+    /**
+     * Constructor to create Event object.
+     *
+     * @param description specific details of Event.
+     * @param date when the Event is taking place.
+     */
+    public Event(String description, String date) {
+        super(description);
+        this.date = date;
+    }
+
+    /**
+     * Gets the date of Event.
+     *
+     * @return date of the Event.
+     */
+    public String getDate() {
+        return this.date;
+    }
+
+    private String convertDateAndTime() {
```

Comment:
Both Deadline.java and this file have missing Javadoc comments for only this method within the class. The method name can be a bit vague about what it's actually converting the date/time to, so a quick `return` clarification in the documentation would help a lot.